app/helpers/d3plus.js

```diff
@@ -77,7 +77,7 @@ export default {
     }
   },
   shapeConfig: {
-    hoverOpacity: 0.1,
+    hoverOpacity: 0.7,
```

Comment:
Can you take another look into this value? The reason I had changed it to 0.1 is that it is very hard to tell which shapes are being highlighted in visualizations with slightly transparent fills (http://localhost:3300/profile/040AF00182#Conditions).

Maybe this is something where we can selectively adjust this value depending on the visualization?